modules/tbs/data/object_prototypes/title_controller_base.cfg

```diff
@@ -43,8 +43,8 @@
 		[
 			if(is_offline = false, [
 				spawn('label', {
-					x: lib.gui.py(130),
-					mid_y: lib.gui.py(76),
+					x: lib.gui.py(140),
+					mid_y: lib.gui.py(80),
```

Comment:
I would never had come up with this idea... it's a small detail change, but it certainly looks better.